build/pack.c

```diff
@@ -14,6 +14,8 @@
 #include <rpm/rpmfileutil.h>
 #include <rpm/rpmlog.h>
 
+#include <prtpool.h>			/* NSPR thread pools */
```

Comment:
rpm already has a hard dependency on nss, and nss has a hard dependency on nspr, so it seemed obvious to utilize the thread pool API from nspr. Why you are reluctant?